admin/commands/management/add_mysql.go

```diff
@@ -89,14 +89,15 @@ func (res *addMySQLResult) TablestatStatus() string {
 
 // AddMySQLCommand is used by Kong for CLI flags and commands.
 type AddMySQLCommand struct {
-	ServiceName   string `name:"name" arg:"" default:"${hostname}-mysql" help:"Service name (autodetected default: ${hostname}-mysql)"`
-	Address       string `arg:"" optional:"" help:"MySQL address and port (default: 127.0.0.1:3306)"`
-	Socket        string `help:"Path to MySQL socket"`
-	NodeID        string `help:"Node ID (default is autodetected)"`
-	PMMAgentID    string `help:"The pmm-agent identifier which runs this instance (default is autodetected)"`
-	Username      string `default:"root" help:"MySQL username"`
-	Password      string `help:"MySQL password"`
-	AgentPassword string `help:"Custom password for /metrics endpoint"`
+	ServiceName       string `name:"name" arg:"" default:"${hostname}-mysql" help:"Service name (autodetected default: ${hostname}-mysql)"`
+	Address           string `arg:"" optional:"" help:"MySQL address and port (default: 127.0.0.1:3306)"`
+	Socket            string `help:"Path to MySQL socket"`
+	NodeID            string `help:"Node ID (default is autodetected)"`
+	PMMAgentID        string `help:"The pmm-agent identifier which runs this instance (default is autodetected)"`
+	Username          string `help:"MySQL username"`
+	Password          string `help:"MySQL password"`
+	AgentPassword     string `help:"Custom password for /metrics endpoint"`
+	CredentialsSource string `help:"Credentials provider"`
```

Comment:
Can we please improve the help text here and in other instances of `CredentialsSource`?
It's not clear what value is expected here.

Reply:
Done.